### DISCUSSION TITLE
AWS support on CI/CD

### DISCUSSION THREAD

**Discussion by iwoork** [mkosir/frontend-monorepo-boilerplate#4] [Ideas]
I love this boilerplate and have used it in one of my project. One thing I've noticed is that the github action is deeply tied to to netlify. Perhaps there should be an example to get this working with AWS?

**Comment by mkosir** [comment]
Hey @iwoork 👋 , glad you find it useful!
Went with one of the hosting providers that offers free tier for proof of concept (Netlify in this case), but I think the deployment step should be really similar regardless of the provider.
Feel free to submit PR, agree it would be nice to see more examples in CI/CD script, perhaps we can comment out rest of the providers, so consumer of boilerplate easily decides which one to use 👍
